components/tooltip/__tests__/tooltip.test.tsx

```
@@ -590,6 +590,41 @@
     expect(container).toMatchSnapshot();
   });
 
+  it('should not render content initially when forceRender=false', async () => {
+    const { container } = render(
+      <Tooltip title="forceRender=false" forceRender={false}>
+        <button>forceRender=false</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)